app/models/PlayExampleProjects.scala

```diff
@@ -33,14 +32,14 @@ class ExamplesModule extends AbstractModule {
 @Singleton
 class PlayExampleProjectsService @Inject()(configuration: Configuration,
                                            ws: WSClient,
-                                           cache: CacheApi)(implicit ec: ExecutionContext) {
+                                           cache: SyncCacheApi)(implicit ec: ExecutionContext) {
   import scala.collection.JavaConverters._
 
   private val logger = org.slf4j.LoggerFactory.getLogger(this.getClass)
 
-  val templatesUrl = configuration.getString("examples.apiUrl").get
+  val templatesUrl: String = configuration.get[String]("examples.apiUrl")
 
-  val validPlayVersions: Set[String] = configuration.getStringList("examples.playVersions").get.asScala.toSet
+  val validPlayVersions: Set[String] = configuration.underlying.getStringList("examples.playVersions").asScala.toSet
```

Comment:
Hum, the deprecation message is suggesting what I've did:

https://github.com/playframework/playframework/blob/4ebc9d670cf76a1c0463aa3b6b9477c31d13c3bf/framework/src/play/src/main/scala/play/api/Configuration.scala#L895-L895

We should change it both here and there then.